descriptor.md

```diff
@@ -61,22 +61,30 @@ The _digest_ property of a Descriptor acts as a content identifier, enabling [co
 It uniquely identifies content by taking a [collision-resistant hash](https://en.wikipedia.org/wiki/Cryptographic_hash_function) of the bytes.
 If the identifier can be communicated in a secure manner, one can retrieve the content from an insecure source, calculate the digest independently, and be certain that the correct content was obtained.
 
-The value of the digest property, the _digest string_, is a serialized hash result, consisting of an _algorithm_ portion and a _hex_ portion.
-The algorithm identifies the methodology used to calculate the digest; the hex portion is the lowercase hex-encoded result of the hash.
+The value of the digest property, the _digest string_, is a serialized hash result, consisting of an _algorithm_ portion and an _encoded_ portion.
+The algorithm identifies the methodology used to calculate the digest and serialize the _encoded_ portion.
 
 The digest string MUST match the following grammar:
 
 ```
-digest      := algorithm ":" hex
-algorithm   := /[a-z0-9_+.-]+/
-hex         := /[a-f0-9]+/
+digest      := algorithm ":" encoded
+algorithm   := /[a-z0-9]+(?:[+._-][a-z0-9]+)*/
+encoded     := /[a-zA-Z0-9]+/
```

Comment:
> `=` actually isn't needed for base64. I'll add `_-`.

[From the RFC][1]:

> Implementations MUST include appropriate pad characters at the end of encoded data unless the specification referring to this document explicitly states otherwise.

So if you do not allow `=` than all specs that use base 64 will need that explicit statement.  This spec doesn't currently reference RFC 4648, so we don't need that explicit statement, but I see no reason to forbid `=` in the `encoded` part.

[1]: https://tools.ietf.org/html/rfc4648#section-3.2